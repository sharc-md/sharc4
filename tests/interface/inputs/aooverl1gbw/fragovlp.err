/bin/sh: orca_fragovl: command not found
