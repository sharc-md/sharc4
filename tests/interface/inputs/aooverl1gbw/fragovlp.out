Entering ORCA_FRAGOVL

Reading 1st GBW file    ... /user/pitesa/sharc_main/tests/interface/inputs/aooverl1gbw/ORCA.gbw


WARNING!!!!!!!
Your GBWFile is either corrupt or from a different ORCA version!
Please be VERY careful with your calculation results!!!

Data_size, sizeof( TOrcaInfo ): 651728, 871248
[file orca_tools/Tool-GTO-Bases/basissets.cpp, line 823]: Wrong number of basis-sets stored!

